dask/base.py

```diff
@@ -717,13 +717,13 @@ def normalize_function(func):
 
 
 def _normalize_function(func):
-    if isinstance(func, curry):
+    if isinstance(func, curry) and hasattr(func, "_partial"):
         func = func._partial
     if isinstance(func, Compose):
         first = getattr(func, "first", None)
         funcs = reversed((first,) + func.funcs) if first else func.funcs
         return tuple(normalize_function(f) for f in funcs)
-    elif isinstance(func, partial):
+    elif isinstance(func, (partial, curry)):
```

Comment:
Same questions here, though it may be related to the other change made in this module

Reply:
See comment above.  I believe this is a bug in toolz, which I will be submitting a PR shortly to fix.

Reply:
We can check for curry here because curry and partial have the same `func`, `args`, and `keywords` attributes.